redfish/provider_test.go

```diff
@@ -0,0 +1,13 @@
+package redfish
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+var testAccProviders map[string]*schema.Provider
+var testAccProvider *schema.Provider
+
```

Comment:
Hey @grantcurell 
I propose the following. Why don't we define an struct lile:
~~~
type TestingServerCredentials struct {
	Username string
	Password string
	Endpoint string
	Insecure bool
}
~~~
And we create a var under *testAccProvider* like:
~~~
var credentials *TestingServerCredentials
~~~
So we can init that at the init() func, and grab the actual values from ENV vars to make test fit every single scenario (and also to facilitate us to integrate it in our future CI/CD pipeline)
~~~
func init() {
	testAccProvider = Provider()
	testAccProviders = map[string]*schema.Provider{
		"redfish": testAccProvider,
	}
	
	credentials = &TestingServerCredentials{
		Username: os.Getenv("TF_TESTING_USERNAME"),
		Password: os.Getenv("TF_TESTING_PASSWORD"),
		Endpoint: os.Getenv("TF_TESTING_ENDPOINT"),
		Insecure: false,
	}
} 
~~~